test/BaselineOutput/Common/EntryPoints/core_ep-list.tsv

```diff
@@ -54,7 +54,7 @@ Trainers.FastTreeBinaryClassifier	Uses a logit-boost boosted tree learner to per
 Trainers.FastTreeRanker	Trains gradient boosted decision trees to the LambdaRank quasi-gradient.	Microsoft.ML.Runtime.FastTree.FastTree	TrainRanking	Microsoft.ML.Runtime.FastTree.FastTreeRankingTrainer+Arguments	Microsoft.ML.Runtime.EntryPoints.CommonOutputs+RankingOutput
 Trainers.FastTreeRegressor	Trains gradient boosted decision trees to fit target values using least-squares.	Microsoft.ML.Runtime.FastTree.FastTree	TrainRegression	Microsoft.ML.Runtime.FastTree.FastTreeRegressionTrainer+Arguments	Microsoft.ML.Runtime.EntryPoints.CommonOutputs+RegressionOutput
 Trainers.FastTreeTweedieRegressor	Trains gradient boosted decision trees to fit target values using a Tweedie loss function. This learner is a generalization of Poisson, compound Poisson, and gamma regression.	Microsoft.ML.Runtime.FastTree.FastTree	TrainTweedieRegression	Microsoft.ML.Runtime.FastTree.FastTreeTweedieTrainer+Arguments	Microsoft.ML.Runtime.EntryPoints.CommonOutputs+RegressionOutput
-Trainers.FieldAwareFactorizationMachineBinaryClassifier	Train a field-aware factorization machine for binary classification	Microsoft.ML.Runtime.FactorizationMachine.FieldAwareFactorizationMachineTrainer	TrainBinary	Microsoft.ML.Runtime.FactorizationMachine.FieldAwareFactorizationMachineTrainer+Arguments	Microsoft.ML.Runtime.EntryPoints.CommonOutputs+BinaryClassificationOutput
+Trainers.FieldAwareFactorizationMachineBinaryClassifier	Train a field-aware factorization machine for binary classification	Microsoft.ML.Runtime.Recommend.FieldAwareFactorizationMachineTrainer	TrainBinary	Microsoft.ML.Runtime.Recommend.FieldAwareFactorizationMachineTrainer+Arguments	Microsoft.ML.Runtime.EntryPoints.CommonOutputs+BinaryClassificationOutput
```

Comment:
>FieldAwareFactorizationMachineBinaryClassifier [](start = 9, length = 46)

Can I point on a fact what it shouldn't happen? Why FFM change it namespace if you work on MatrixFactorization?

Reply:
It was an accident, I will rebuild the APIs once everything is finished. Thanks.

---
In reply to: [226503004](https://github.com/dotnet/machinelearning/pull/1263#discussion_r226503004) [](ancestors = 226503004)